src/msg/earn/index.ts

```diff
@@ -1,16 +1,24 @@
-import { Coin } from '../../types/Coin';
+import { Strategy, Coin } from '../../types';
 
-function newMsgDeposit(depositor: string, amount: Coin[]) {
+function newMsgDeposit(depositor: string, amount: Coin[], strategy: Strategy) {
```

Comment:
Confirmed with Derrick that `amount` should be a singular `Coin` in both functions.

https://kava-network.slack.com/archives/C03UN2CK678/p1661281961599829